.mise/tasks/ci/autofix/lint/xtask-reuse/all

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+# MISE description="Run xtask check reuse compliance on all files"
+
+set -e
+
+if test ! -d "target" ; then
```

Comment:
Why is this conditional on a target directory? Why can't this be run after cargo build?

Reply:
In that case, perhaps the script should print a message instead of silently passing? Otherwise the presence of the directory would imply that all is well, and if that happens by accident in the CI (because of the cache or an innocently placed `cargo build` in an earlier yaml instruction), then that would not be quite so good and hard to diagnose.

I'm inclined to say that it would be better to actually fail with a message and let the user clean up their `target`.